### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by hieunguyen64**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,27 +15,33 @@
...
             </div>
-            <div class="addtocart col-lg-3 col-sm-6">
+            <div class="addtocart col-6 col-sm-3 col-md-3 col-lg-3">
                 <form class="shopping-cart">
                     <button onclick="Comingsoon()" type="button"><i class="fa fa-shopping-cart"></i></button>
-                    <label>Shopping cart</label>
+                    <label style="padding-top: 10px; ">Cart</label>
+                </form>
+            </div>
+            <div class="search-container col-sm-12  col-md-6 col-lg-7 d-md-none d-block">
```
Comment (thienhophu) [inline index.html:35]: remove unused space

File: requirement/.~lock.ProjectEstimate.excel# [hunk 2]
```diff
@@ -1 +1 @@
-,hieunguyen,hieunguyen-R580-R590,10.04.2020 09:55,file:///home/hieunguyen/.config/libreoffice/4;
\ No newline at end of file
+,hieunguyen,hieunguyen-R580-R590,12.04.2020 08:47,file:///home/hieunguyen/.config/libreoffice/4;
```
Comment (thienhophu) [inline requirement/.~lock.ProjectEstimate.excel#:1]: add this file to .gitignore

File: index.html (hunk 1 above)
Comment (hieunguyen64) [reply index.html:35]: done

File: requirement/.~lock.ProjectEstimate.excel# (hunk 2 above)
Comment (hieunguyen64) [reply requirement/.~lock.ProjectEstimate.excel#:1]: done

**Pull request merged**